service/controller/app/v1/resource/configmap/create.go

```diff
@@ -47,15 +47,15 @@ func (r *Resource) newCreateChange(ctx context.Context, currentResource, desired
 		return nil, microerror.Mask(err)
 	}
 
-	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding out if the %#q configmap has to be created", desiredConfigMap.Name))
+	r.logger.LogCtx(ctx, "level", "debug", "message", "finding out if the configmap has to be created")
```

Comment:
Simplifies the logging. The configmap will be nil if no config was specified in the app CR.